src/utils.jl

```diff
@@ -0,0 +1,50 @@
+function axis_types(wcs::WCSTransform)
+    result = []
+    for axis in 1:wcs.naxis
+        subresult = ""
+        # coordinate type
+        if wcs.ctype[axis][1:6] == "STOKES"                                                   # stokes
+            subresult += "1"
+        elseif wcs.ctype[axis][1:2] in ["RA", "DEC"] || wcs.ctype[axis][1:8] == "CUBEFACE"    # celestial
+            subresult += "2"
+        elseif wcs.ctype[axis][1:4] in ["FREQ", "ENER", "WAVN", "VRAD", "WAVE", "VOPT", "ZOPT", "AWAV", "VELO", "BETA"]
+            subresult += "3"                                                                  # spectral
+        else
+            subresult += "0"			                                                      # others
+        end    
+
+        # axis type
+        if wcs.ctype[axis][6:8] == "---"
+            subresult += "0"
+        elseif wcs.ctype[axis][1:6] == "STOKES" || wcs.ctype[axis][1:8] == "CUBEFACE"
+            subresult += "1"
+        elseif wcs.ctype[axis][6:8] in ["F2W", "F2V", "F2A", "W2F", "W2V", "W2A", "V2F", "V2W", "V2A", "A2F", "A2W", "A2V"]
+            subresult += "3"
+        elseif wcs.ctype[axis][6:8] != "---" && subresult[1] == "2"
+            subresult += "2"
+        elseif wcs.ctype[axis][6:8] == "LOG"
+            subresult += "4"
+        else
+            subresult += "5"
+        end
+
+        # group no.
```

Comment:
I'm not sure about this part. As per my understanding `GCOUNT` field in Header carry this information but in WCS.jl there is no such variable to denote this (or maybe another name is used).